jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -176,6 +176,20 @@
                 export MULTIZONE="true"
                 export E2E_ZONES="us-central1-a us-central1-b us-central1-f" # Where the nodes reside.  Master is in the first one.
                 export KUBE_GCE_ZONE="us-central1-a" # Where the master resides - hangover due to legacy scripts.
+        - 'gce-federation':
+            description: 'Run all federated e2e tests.'
+            timeout: 300
+            emails: 'quinton@google.com, colin.hom@coreos.com'
+            test-owner: 'quinton'
+            job-env: |
+                export PROJECT="k8s-jkns-e2e-gce-ubelite"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Federation\]"
+                export GINKGO_PARALLEL="n" # We don't have namespaces yet in federated apiserver, so we need to serialize
+                export FEDERATED="true"
+                export E2E_ZONES="us-central1-a us-central1-b us-central1-f" # Where the clusters will be created. Federated components are now deployed to the last one.
+                export FEDERATED_PUSH_REPO_BASE=gcr.io/xxxxx # gcr.io/xxxxx/federated-apiserver needs to be a valid repo
```

Comment:
I need help creating a `gcr.io/xxxxx/federated-apiserver` repo. This repo will be pushed to via `gcloud docker push` since it's a gcr.io/* url, so I think the permissions should be easy to get right.